nsqadmin/static/js/collections/nodes.js

```diff
@@ -2,7 +2,7 @@ var Backbone = require('backbone');
 
 var AppState = require('../app_state');
 
-var Node = require('../models/node');
+var Node = require('../models/node'); //eslint-disable-line no-undef
```

Comment:
Not a restricted keyword.

`Nodejs` is a _holy_ word - maybe that's what you're thinking of?